src/subdomains/generic/user/models/user-data/user-data.service.ts

```diff
@@ -222,25 +228,51 @@ export class UserDataService {
       data.organizationCountry = null;
     }
 
-    return this.userDataRepo.save(Object.assign(user, data));
+    for (const user of userData.users) {
+      await this.siftService.updateAccount({
+        $user_id: user.id.toString(),
+        $user_email: data.mail,
+        $name: `${data.firstname} ${data.surname}`,
+        $phone: data.phone,
+        $billing_address: {
+          $name: `${data.firstname} ${data.surname}`,
+          $address_1: data.street,
+          $city: data.location,
+          $phone: data.phone,
+          $country: country.name,
+          $zipcode: data.zip,
+        },
+        kyc_level: userData.kycLevel,
```

Comment:
Doesn't really make sense to update the KYC level here 🤔 let's leave it away for now.